mulhome_scripts/build_push_wave.py

```diff
@@ -18,15 +21,19 @@ def build_push_wave():
     config = configparser.ConfigParser()
     config.read(INI_PATH)
     sys.path.append(jupiter_config.WAVE_PATH)
-    print(jupiter_config.WAVE_PATH)
+    logging.debug(jupiter_config.WAVE_PATH)
 
     import wave_docker_files_generator as dc 
     os.chdir(jupiter_config.WAVE_PATH)
 
+    logging.debug(jupiter_config.APP_NAME)
+
     home_file = dc.write_wave_home_docker(app_file = jupiter_config.APP_NAME, ports = jupiter_config.FLASK_DOCKER)
     worker_file = dc.write_wave_worker_docker(app_file = jupiter_config.APP_NAME, ports = jupiter_config.FLASK_DOCKER)
 
 
+    print(home_file)
```

Comment:
Should these be logging calls? Also, useful information on the progress of the program should be of type INFO not debug. Debug is only turned on when someone is trying to figure out why something has broken. Particularly after seeing a WARN or ERROR.

Reply:
I missed this. It can be info.